tests/e2e/test_execute_async.py

```diff
@@ -1,50 +1,62 @@
 from tests.e2e.test_driver import PySQLPytestTestCase
 
-from databricks.sql.ae import AsyncExecutionStatus as AsyncExecutionStatus
-
+from databricks.sql.ae import (
+    AsyncExecutionStatus,
+    AsyncExecutionException,
+    AsyncExecution,
+)
+import pytest
 import time
 
-LONG_RUNNING_QUERY =  """
+LONG_RUNNING_QUERY = """
 SELECT SUM(A.id - B.id)
 FROM range(1000000000) A CROSS JOIN range(100000000) B 
 GROUP BY (A.id - B.id)
 """
 
+
 class TestExecuteAsync(PySQLPytestTestCase):
+    @pytest.fixture
+    def long_running_ae(self, scope="function") -> AsyncExecution:
```

Comment:
For simplicity I added this pytest fixture so I don't need to duplicate this code for my "raises an exception" assertions.